### DISCUSSION TITLE
Some words are widely accepted and normal, definitely not offensive

### DISCUSSION THREAD

**Discussion by PeterH220** [retextjs/retext#69] [General] [answered]
There is lot of terms that describe nationalities, medical terms, politically correct and "sanitized" terms and absolutely normal words that shouldn't be on the list. Just because they might be used as offensive doesn't mean that they should be listed as such.

https://github.com/retextjs/retext-profanities

adult, africa, african, allah, amateur, american, angie, arab, arabs, areola, asian, australian, babe, babies, backseat, banana, bananas, beast, beaver, bible, bigger, black, brea5t, breast, canadian, catholic, catholics, cemetery, childrens, chinese, christ, christian, church, cigarette, cigs, coconut, coconuts, coitus, color, colored, coloured, conservative, copulate, creamy, deposit, desire, ejaculate, ejaculated, ejaculating, ejaculation, erect, erection, ethiopian, ethnic, european, fairies, fairy, faith, filipina, filipino, fire, firing, genital, german, gin, girls, god, headlights, heterosexual, hole, israel, israeli, israels, italiano, itch, japanese, jesus, jesuschrist, jewish, joint, kid, kink, kinky, knife, laid, latin, liberal, libido, lingerie, liquor, mexican, minority, mormon, naked, narcotic, nipple, osama, pee, porn, porno, poverty, primetime, pros, prostitute, protestant, pube, pubic, que, rearend, rearentry, rectum, refugee, remains, republican, sex, sexing, sextoy, sextoys, sexual, sexy, sos, soviet, spa, spade, spades, stroke, sweetness, testicle, testicles, threesome, threeway, tnt, toilet, triplex, uk, urinary, urinate, urine, uterus, vagina, vaginal, vatican, vibr, vibrater, vibrator, vietcong, virgin, vulva, womens

**Comment by wooorm** [comment, marked as answer]
You can pass `sureness: 1` or ignore the terms you don’t want to `retext-profanities`
